src/oxygen/__init__.py

```diff
@@ -1,10 +1,8 @@
 from .base_handler import BaseHandler
 from .oxygen import listener, OxygenLibrary
-from .robot_interface import RobotInterface
+from .robot_interface import RobotInterface, get_keywords_from
 from .version import VERSION
 
-__all__ = ['BaseHandler', 'listener', 'OxygenLibrary', 'RobotInterface']
+__all__ = ['BaseHandler', 'listener', 'OxygenLibrary', 'RobotInterface',
+           'get_keywords_from']
```

Comment:
`__all__` should be used to declare the public interface of the module. `get_keywords_from` is not intended to be called from outside of Oxygen, so it should not be defined in `__all__`. Nor should `RobotInterface` for that matter; the old code was wrong as well.